authority/ssh.go

```diff
@@ -268,6 +294,24 @@ func (a *Authority) SignSSH(ctx context.Context, key ssh.PublicKey, opts provisi
 		)
 	}
 
+	// Send certificate to webhooks for authorization
+	if webhookCtl != nil {
+		whAuthBody, err := webhook.NewRequestBody(
+			webhook.WithSSHCertificate(certificate, certTpl),
+		)
+		if err != nil {
+			return nil, errs.ApplyOptions(
+				errs.ForbiddenErr(err, "authority.SignSSH: error signing certificate"),
+			)
+		}
+		err = webhookCtl.Authorize(whAuthBody)
+		if err != nil {
+			return nil, errs.ApplyOptions(
+				errs.ForbiddenErr(err, "authority.SignSSH: error signing certificate"),
+			)
+		}
+	}
+
```

Comment:
This feels natural here, but you might want to extract this to a new function.

Reply:
10399d92